pkg/metrics/statsd/e2e/metrics.go

```diff
@@ -0,0 +1,56 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+)
+
+const (
+	DimensionARMGeoLocation = "armGeoLocation"
+	DimensionARMResourceID  = "armResourceID"
+	DimensionResourceType   = "resourceType"
+	DimensionSucceeded      = "Succeeded"
+
+	LogEntryIsE2EEmittableMetric = "IsE2EEmittableMetric"
+	LogEntryMetricName           = "MetricName"
+	LogEntryMetricStatus         = "MetricStatus"
```

Comment:
```suggestion
	LogEntryMetricSuccess         = "MetricSuccess"
```
this or `isSuccessful` or something that can be represented by a bool